docs/clients/react-native.md

```diff
@@ -1,3 +1,154 @@
-Documentation for Textile's React Native SDK (react-native-sdk) can be found at https://textileio.github.io/react-native-sdk.
+Textile's React Native SDK is designed to work on iOS and Android devices in app's built with [React Native](https://facebook.github.io/react-native/).
 
-NPM package: https://www.npmjs.com/package/@textile/react-native-sdk
+## Installation
+
+The React Native SDK is published as an [NPM Package](https://www.npmjs.com/package/@textile/react-native-sdk). You are free to use the [source code](https://github.com/textileio/react-native-sdk) directly or use a package manager to install the official release.
+
+#### NPM
+
+```JavaScript
+npm install @textile/react-native-sdk
```

Comment:
Think you need a `--save` or something. Can't remember.

Reply:
old npm, no longer needed